README.md

```diff
@@ -21,6 +21,21 @@ Image Inspector can extract docker images to a target directory and
     Coll:   lost+found                          4096  Dec 10 20:24
     ...
 
+Image Inspector can inspect images using OpenSCAP and serve the scan result.
+The OpenSCAP scan report will be served on <serve_path>/api/v1/openscap and 
+the status of the scan will be available on <serve_path>/api/v1/metadata in
+the OpenSCAP section.
+
+    $ sudo ./image-inspector --image=fedora:22 --path=/tmp/image-content --scan-type=openscap
+			--serve 0.0.0.0:8080 --chroot
+    2016/05/25 16:12:04 Image fedora:22 is available, skipping image pull
+    2016/05/25 16:12:04 Extracting image fedora:22 to /tmp/image-content
+    2016/05/25 16:12:14 OpenSCAP scanning /tmp/image-content. 
+						Placing results in /var/tmp/image-inspector-scan-results-845509636
+    2016/05/25 16:12:20 Serving image content /tmp/image-content on 
+						webdav://0.0.0.0:8080/api/v1/content/
```

Comment:
@enoodle can you remove the long white-spaces indentation for "Placing results in..." and "webdav://0.0.0.0...". (Or maybe there's a reason for it?) Thanks.